packages/caravan-health/src/waste.ts

```diff
@@ -128,44 +129,46 @@ export class WasteMetrics extends WalletMetrics {
   /*
     Name : calculateDustLimits
     Definition : 
-      "Dust" is defined in terms of dustRelayFee,
-      which has units satoshis-per-kilobyte.
-      If you'd pay more in fees than the value of the output
-      to spend something, then we consider it dust.
-      A typical spendable non-segwit txout is 34 bytes big, and will
-      need a CTxIn of at least 148 bytes to spend:
-      so dust is a spendable txout less than
-      182*dustRelayFee/1000 (in satoshis).
-      546 satoshis at the default rate of 3000 sat/kB.
-      A typical spendable segwit txout is 31 bytes big, and will
-      need a CTxIn of at least 67 bytes to spend:
-      so dust is a spendable txout less than
-      98*dustRelayFee/1000 (in satoshis).
-      294 satoshis at the default rate of 3000 sat/kB.
+      Dust limits are the limits that help to determine the lower and upper limit of the UTXO 
+      that can be spent economically.
+      The lower limit is below which the UTXO will actually behave as a dust output and the 
+      upper limit is above which the UTXO will be safe and economical to spend.
 
     Calculation :
       lowerLimit - Below which the UTXO will actually behave as a dust output.
       upperLimit - Above which the UTXO will be safe and economical to spend.
       riskMultiplier - A factor that helps to determine the upper limit.
```

Comment:
Can you leave a better definition for "riskMultiplier". How does it determine the upper limit? why is it called a risk multiplier? How should a user of this api think about what to set as this value? why does it default to 2?

Reply:
Sure, I will